### ISSUE TITLE
META - Pending Pull Requests

### ISSUE THREAD

**Issue by mliu49** [ReactionMechanismGenerator/RMG-Py#886]
There are many pending pull requests (and soon to be made pull requests) right now and some are inter-related, so I wanted a place to discuss the relationships and which ones should be fast-tracked to merging.

#835 - Resonance Part 2
- The second part of my resonance changes, where the main resonance algorithm is actually changed
- Currently pending additional tests (PDD) by Kehang
- Changes will be required once #864 is merged

#857 - Degeneracy fix for identical reactants
- Contains minor change to degeneracy calculation and related unit tests
- Currently pending review by Kehang
- Can be merged without conflict

#864 - Bond and GroupBond store bond order as float
- Changes bond orders from strings to floats
- Can be merged without conflict

Future 1 - Additional degeneracy changes and benzene bond reactivity
- Depends on #835, #857, #864 

Future 2 - Resonance hybrids for symmetry calculation
- Depends on #835, #864, Future 1

I think #857 and #864 should be merged as soon as possible, so that the appropriate changes can be made to #835 and Future 1. Once #835 is merged, then I'll make new pull request(s) for Future 1.

These are the main pull requests that I'm involved with. Please comment if you have other pull requests that are dependent or need to be merged soon.

**Comment by KEHANG** [comment]
seems I'm the limiting step now. I'll see what I can do to speed up the process.

**Comment by mliu49** [comment]
Thanks for your hard work @KEHANG!
